app/components/registration-photos/component.js

```diff
@@ -0,0 +1,13 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+export default class RegistrationPhotosComponent extends Component {
+  @tracked chosenPhoto;
+
+  @action
+  onChosenPhoto(event) {
+    this.chosenPhoto = event.target.value;
+    this.args.setChosenPhoto(this.chosenPhoto);
```

Comment:
if you get date from src it would look like:
```suggestion
  onChosenPhoto(event) {
    this.chosenPhoto = event.target.src
    this.args.setChosenPhoto(this.chosenPhoto);
```